### PR TITLE
Fixturize tests/frame/test_operators.py

### PR DESCRIPTION
**Pull request by h-vetinari**
One more steps towards #22471.

### CODE REVIEW

File: pandas/tests/frame/conftest.py [hunk 1]
```diff
@@ -94,6 +94,9 @@ def mixed_float_frame():
     return df
 
 
+mixed_float_frame2 = mixed_float_frame
```
Comment (WillAyd) [inline pandas/tests/frame/conftest.py]: Do you see this being used elsewhere than the one test? If not probably better just to create a copy directly within the test

File: pandas/tests/frame/test_operators.py [hunk 2]
```diff
@@ -340,95 +341,96 @@ def test_combineFrame(self):
...
-        added_rev = frame_copy + self.frame
+        added_rev = frame_copy + float_frame
         assert np.isnan(added['D']).all()
         assert np.isnan(added_rev['D']).all()
 
         # corner cases
 
         # empty
-        plus_empty = self.frame + self.empty
+        plus_empty = float_frame + DataFrame({})
```
Comment (WillAyd) [inline pandas/tests/frame/test_operators.py]: So we have already a few fixtures for `empty_frame` (one in sparse.frame.conftest and another in the resample conftest). Maybe it makes sense to have one at the highest level conftest instead?

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/tests/frame/test_operators.py (hunk 2 above)
Comment (jreback) [reply pandas/tests/frame/test_operators.py]: we removed empty, this is ok (though should be ``DataFrame()``)

File: pandas/tests/frame/conftest.py (hunk 1 above)
Comment (jreback) [reply pandas/tests/frame/conftest.py]: agreed don't want this here

**Review by jreback — CHANGES_REQUESTED** [review]
change the way empty is written as per comment

File: pandas/tests/frame/conftest.py (hunk 1 above)
Comment (WillAyd) [reply pandas/tests/frame/conftest.py]: Think you missed removal of this in last commit

**Review by WillAyd — CHANGES_REQUESTED** [review]

**Review by WillAyd — APPROVED** [review]
lgtm @jreback

**Comment by h-vetinari** [comment]
Friendly ping. :)

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks

**Pull request merged**